tests/e2e-ui-tests/page-objects/mobile/workflow/main.po.js

```diff
@@ -0,0 +1,75 @@
+var WorkflowPage = function() {
```

Comment:
I think that we may just load some sample steps for purposes of this tests.
Vehicle inspection step it's not the best one for this purposes as it has some custom elements. 
From our point of view this elements doesn't matter at all as Vehicle inspection step is done only for sales demo purposes. We may just have couple different steps to make sure that we support branching etc.

With that in mind I will not invest more time on this particular test. Just FYI